plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -303,6 +305,7 @@
     protected String _password;
     protected String _guid;
     protected String _vCenterAddress;
+    protected Integer _storageNfsVersion;
```

Comment:
There has been a discussion about slowly remove the underscore at the beginning of variables. Could you please remove it? Thanks.